### ISSUE TITLE
Attach Pika window to the menubar

### ISSUE THREAD

**Issue by kud** [superhighfives/pika#98]
## Issue Description
Currently, the Pika window is a standalone window that can be moved around the screen. However, it would be more convenient if the window could be attached to the menubar like the Sip app does. This would allow for easier access to the colorpicker without cluttering up the screen.

## Proposed Solution
The Pika window should be modified to include an option to attach the window to the menubar. When this option is selected, the window should snap to the menubar and become a dropdown menu. This dropdown menu should include all of the current functionality of the standalone window, including the ability to select colors and copy color codes.

## Acceptance Criteria
-  The Pika window has a new option to attach to the menubar
-  When the option is selected, the window snaps to the menubar and becomes a dropdown menu
-  The dropdown menu includes all of the current functionality of the standalone window

**Comment by superhighfives** [comment]
Yeah, big fan of this idea. I'd love to get around to it, but in the interim if anyone wants to take a pass I'd love to check it out. 👍 Added a help wanted tag, too.
